### ISSUE TITLE
Pihole - api.spotify.com spam

### ISSUE THREAD

**Issue by AFCM1** [raywo/MMM-NowPlayingOnSpotify#89]
Hello,

I have this module running fine on my Rpi. This same Rpi is also hosting a Pihole instance.
[Few days ago, I noticed that my Pihole query logs was filled with queries from api.spotify.com](https://i.imgur.com/RDDAZPa.png)  
  
Is that a normal behaviour? If not what should I do to stop these API spams?

**Comment by Fabrizz** [comment]
Yeah, depending on how you configured it it reaches Spotify X times per minute to fetch the current playback. [Spotify Api Endpoint](https://developer.spotify.com/documentation/web-api/reference/#endpoint-get-information-about-the-users-current-playback)

**Comment by mattbruman** [comment]
> Yeah, depending on how you configured it it reaches Spotify X times per minute to fetch the current playback. [Spotify Api Endpoint](https://developer.spotify.com/documentation/web-api/reference/#endpoint-get-information-about-the-users-current-playback)

Where can I edit how often it pings? Right now by default its twice every second that seems excessive for just needed to know what song is playing.

**Comment by Fabrizz** [comment]
> > Yeah, depending on how you configured it it reaches Spotify X times per minute to fetch the current playback. [Spotify Api Endpoint](https://developer.spotify.com/documentation/web-api/reference/#endpoint-get-information-about-the-users-current-playback)
> 
> Where can I edit how often it pings? Right now by default its twice every second that seems excessive for just needed to know what song is playing.

I don't know if the updatesEvery config option changes the polling rate to spotify or just the dom builder, but try that.

**Comment by DJB1966** [comment]
Did anyone find a fixed problem with api.spotify.com pihole spam I'm having the same problem?

**Comment by l3d00m** [comment]
@DJB1966 set config `updatesEvery: 5` or something, it specifies in seconds how often spotify is fetched. this will make your progress bar jump more though IIRC.
